backend/tests/test_reservation_routes.py

```
@@ -1,8 +1,10 @@
+from datetime import datetime
 import pytest
 from flask import Flask
+from sqlalchemy import text
 from routes.reservation_routes import reservation_bp
 from database import engine, Base, SessionLocal
-from models import Room, Structure
+from models import AdminStructure, Client, ClientReservations, Reservation, Role, Room, Structure, StructureReservationsView, User
```

Ruff (0.8.2)

`models.StructureReservationsView` imported but unused

Remove unused import: `models.StructureReservationsView`

(F401)